### DISCUSSION TITLE
File.yaml  load on app android nebula

### DISCUSSION THREAD

**Discussion by luisnodealert** [slackhq/nebula#890] [Ideas]
I would like to ask you developers if it is possible to load the .yaml file directly to load keys and settings quickly.
it would be great to configure android nebula app quickly .. or do a restore very quickly

**Comment by IanVS** [comment]
I think this is better tracked under https://github.com/DefinedNet/mobile_nebula/issues/135.  @luisnodealert let me know if you mean something different here.

**Comment by luisnodealert** [comment]
ok thanks it's my same request but I've got the wrong place.
we can close this discussion
